packages/eslint-plugin/tests/rules/adjacent-overload-signatures.test.ts

```diff
@@ -5,924 +5,924 @@ import rule from '../../src/rules/adjacent-overload-signatures';
 const ruleTester = new RuleTester();
 
 ruleTester.run('adjacent-overload-signatures', rule, {
-  valid: [
+  invalid: [
```

Comment:
yeah I definitely don't like this.
I think that the standard of valid, invalid is better.
I'm fine with everything else being sorted -- but I think this top-level would be best left as is.